torchao/kernel/autotuner.py

```diff
@@ -0,0 +1,218 @@
+import torch
+import pathlib
+import os
+import triton
+import pickle
+import logging
+
+
+def do_bench_triton(fn, warmup=25, rep=100, grad_to_none=None, quantiles=None, fast_flush=True, return_mode="mean"):
+    assert return_mode in ["min", "max", "mean", "median"]
+    import torch
+    """
+    Benchmark the runtime of the provided function. By default, return the median runtime of :code:`fn` along with
+    the 20-th and 80-th performance percentile.
+
+    :param fn: Function to benchmark
+    :type fn: Callable
+    :param warmup: Warmup time (in ms)
```

Comment:
warmup is not a time it seems to be an int so not in ms

Reply:
Why can't milliseconds be given in int?

Reply:
I think this is just saying "run warmup until at least 25ms were spent"